griptape/drivers/sql/snowflake_sql_driver.py

```diff
@@ -53,18 +53,21 @@ def execute_query_raw(self, query: str) -> list[dict[str, Optional[Any]]]:
         with self.engine.connect() as con:
             results = con.execute(sqlalchemy.text(query))
 
-            if results.returns_rows:
-                return [{column: value for column, value in result.items()} for result in results]
+            if results is not None:
+                if results.returns_rows:
+                    return [{column: value for column, value in result.items()} for result in results]
+                else:
+                    raise ValueError("No results found")
             else:
-                return None
+                raise ValueError("No results found")
 
     def get_table_schema(self, table: str, schema: Optional[str] = None) -> Optional[str]:
         sqlalchemy = import_optional_dependency("sqlalchemy")
 
         try:
             metadata_obj = sqlalchemy.MetaData()
             metadata_obj.reflect(bind=self.engine)
-            table = sqlalchemy.Table(table, metadata_obj, schema=schema, autoload=True, autoload_with=self.engine)
-            return str([(c.name, c.type) for c in table.columns])
+            table_name = sqlalchemy.Table(table, metadata_obj, schema=schema, autoload=True, autoload_with=self.engine)
+            return str([(c.name, c.type) for c in table_name.columns])
```

Comment:
Revert to `table`. I meant to try `table.__table__.columns`

Reply:
Doesn't solve, #ignoring